lib/scalaplugin.py

```diff
@@ -53,7 +53,7 @@ def get_bloop_artifacts():
 def bloop_classpath(coursier, cache, offline=True):
     deps = get_bloop_artifacts()
     offlineArgs = ["-m", "offline"] if offline else []
-    cmd = [coursier, "fetch"] + offlineArgs + ["--classpath", "--cache", cache] + deps
+    cmd = [coursier, "fetch", "-q"] + offlineArgs + ["--classpath", "--cache", cache] + deps
```

Comment:
I think this is fine (because it's not clear if it's ever useful to see this), but what if instead we changed the wake job verbosity to require verbose to see this output?

Reply:
Coursier outputs these log lines on stderr, does wake control verbosity for stderr?

Reply:
Yeah, was about to add a similar comment. To be precise, I think @jackkoenig and I mean adding `setPlanStdout logVerbose` or `setPlanStdout logDebug`.

Reply:
to get around that we could pipe stderr into stdout, but then we'd need verbose to see any real errors

Reply:
`| setPlanStderr logVerbose` will do it I think,

Reply:
I've tried combo's of piping / setPlanStderr / setPlanStdout, I don't see a way to preserve these log lines without hiding real errors

Reply:
Alright, then this approach is probably best, I'll suggest to coursier they make it possible to send the downloads to stdout

Reply:
I don't think there's a way to do that without writing some extra wrapper code, since the stdout/stderr echoing is independent of the error handling. You can always manually check for errors and print it out by doing something like:

```wake
def plan =
  makePlan cmd visible
  | setPlanStdout logVerbose

def job = runJob plan

match job.getJobOutputs
  Pass _ = # Do whatever the successful case is
  Fail e =
    print "Job failed:\n {job.getJobStdout}"
    Fail e
```

Reply:
I can see how that could work, that should be a global helper function

I've filed https://github.com/sifive/wake/issues/489